camayoc/types/settings.py

```diff
@@ -37,6 +37,12 @@ class OpenShiftOptions(BaseModel):
     nodes: list[str]
 
 
+class AnsibleOptions(BaseModel):
```

Comment:
Why add this class? It doesn't seem to be referenced anywhere, yet.

I think `AnsibleCredentialOptions` should cover all our needs.

Reply:
You're right, I though it would be like a top-level configuration (same as vcenter or openshift), but now I understand that we can configure and use credentials, sources and scans.